### PR TITLE
Ele 1707 add source freshness as test part 2

### PR DESCRIPTION
**Pull request by NoyaArie**

### CODE REVIEW

File: elementary/clients/dbt/slim_dbt_runner.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ def __init__(
         self.adapter_name: Optional[str] = None
         self.connections_manager: Optional[BaseConnectionManager] = None
         self.project_parser: Optional[ManifestLoader] = None
-        self.manifest = None
+        self.manifest: Optional[Manifest] = None
```
Comment (NoyaArie) [inline elementary/clients/dbt/slim_dbt_runner.py:100]: not related - CI error fix

File: elementary/monitor/api/tests/tests.py [hunk 2]
```diff
@@ -37,7 +36,6 @@ def __init__(
     ):
         super().__init__(dbt_runner)
         self.tests_fetcher = TestsFetcher(dbt_runner=self.dbt_runner)
-        self.invocations_api = InvocationsAPI(dbt_runner)
```
Comment (NoyaArie) [inline elementary/monitor/api/tests/tests.py:40]: cleanup

File: elementary/monitor/fetchers/source_freshnesses/schema.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
+from typing import List, Optional
+
+from pydantic import validator
+
+from elementary.utils.schema import ExtendedBaseModel
+from elementary.utils.time import convert_partial_iso_format_to_full_iso_format
+
+
+class SourceFreshnessResultDBRowSchema(ExtendedBaseModel):
+    __test__ = False  # Mark for pytest - The class name starts with "Test" which throws warnings on pytest runs
```
Comment (erikzaadi) [inline elementary/monitor/fetchers/source_freshnesses/schema.py:10]: Is this needed here?

**Review by erikzaadi — APPROVED** [review]
1 small nit, besides that:

**Pull request merged**